crates/red_knot_python_semantic/src/types/class_base.rs

```diff
@@ -169,6 +169,9 @@ impl<'db> ClassBase<'db> {
                 KnownInstanceType::OrderedDict => {
                     Self::try_from_type(db, KnownClass::OrderedDict.to_class_literal(db))
                 }
+                KnownInstanceType::TypedDict => {
+                    Self::try_from_type(db, KnownClass::Dict.to_class_literal(db))
```

Comment:
If we did this instead:

```suggestion
                    Self::try_from_type(db, todo_type!("TypedDict"))
```

 would it get rid of the new false positives we see in the primer report such as

```diff
+ error[lint:unresolved-attribute] /tmp/mypy_primer/projects/hydra-zen/src/hydra_zen/structured_configs/_utils.py:262:5: Type `Literal[DataclassOptions]` has no attribute `__required_keys__`
+ error[lint:unresolved-attribute] /tmp/mypy_primer/projects/hydra-zen/src/hydra_zen/structured_configs/_utils.py:262:42: Type `Literal[DataclassOptions]` has no attribute `__optional_keys__`
```

?

Reply:
that makes sense to me